manager2/src/app/project/project.component.html

```diff
@@ -163,10 +171,16 @@ <h3>Project overview</h3>
               <td>{{selectedProject.path}}</td>
             </tr>
             <tr>
-              <td><b>Size</b>&nbsp;
-                <span class="glyphicon glyphicon-question-sign" title="Storage quota" tooltip></span>
-              </td>
-              <td>{{selectedProject.size}} Gb</td>
+                <td><b>Size</b>&nbsp;
+                    <span class="glyphicon glyphicon-question-sign" title="Storage quota" tooltip></span>
+                </td>
+                <td>{{selectedProject.size}} GB</td>
+            </tr>
+            <tr>
+                <td><b>Cpu</b>&nbsp;
+                    <span class="glyphicon glyphicon-question-sign" title="Storage quota" tooltip></span>
```

Comment:
Not storage quota here

Reply:
fixed: 5ecb403008f78cc3716119dfae809234dba93363